app/(public)/revision/[...revisions]/page.tsx

```
@@ -0,0 +1,126 @@
+    fallbackLang,
+    lastSaved: chaiPage.lastSaved,
+    pageId: chaiPage.id,
+    primaryPageId: chaiPage.primaryPage,
+    languagePageId: chaiPage.languagePageId,
+    pageBaseSlug: chaiPage.slug,
+  };
+  return (
+    <div lang={lang}>
+      <style
+        id="chaibuilder-styles"
+        dangerouslySetInnerHTML={{ __html: pageStyles }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)